packages/office-ui-fabric-react/src/components/Button/Button.test.tsx

```diff
@@ -68,4 +68,31 @@ describe('DefaultButton', () => {
       expect(button.getAttribute('aria-haspopup')).to.equal(null);
     });
   });
+
+  it('Providing onClick and menuProps renders a SplitButton', () => {
```

Comment:
Can you add more tests?

simulating click calls the onClick Callback
simulating enter calls the onclick callback
simulating key down opens/closes the menu. 
Control is disabled, all interactions are noops